packages/create-codes/src/index.ts

```diff
@@ -25,152 +24,21 @@ Create a new codes for front-end app
-        ),
-        default: 0,
-        filter: (val: string) => val.toLowerCase().replace(/\./g, ""),
-      },
-    ])
-  ).jsLibrary;
-
-  const apiSolution = (
-    await inquirer.prompt<{ apiSolution: string }>([
-      {
-        type: "list",
-        name: "apiSolution",
-        message: "Select an API Solution",
-        choices: CLIOptions[jsLibrary].apiSolution,
-        default: "restful",
-      },
-    ])
-  ).apiSolution;
-
-  // TODO: Use modules
-  // eslint-disable-next-line no-unused-vars
-  const useModules = (
-    await inquirer.prompt<{ useModules: string[] }>([
-      {
-        type: "checkbox",
-        name: "useModules",
-        message: "Select module do you want to use",
-        choices: CLIOptions[jsLibrary].useModules,
-      },
-    ])
-  ).useModules;
-
-  const needsTesting = (
-    await inquirer.prompt<{ needsTesting: boolean }>([
-      {
-        type: "confirm",
-        name: "needsTesting",
-        message: "Add Testing codes for Catching bugs early?",
-        default: true,
-      },
-    ])
-  ).needsTesting;
-
-  const needsVitest = (
-    await inquirer.prompt<{ needsVitest?: boolean }>([
-      {
-        type: "confirm",
-        name: "needsVitest",
-        message: "Add Vitest for Unit Testing?",
-        default: true,
-        when: () => needsTesting,
-      },
-    ])
-  ).needsVitest;
-
-  const needsStorybook = (
-    await inquirer.prompt<{ needsStorybook?: boolean }>([
-      {
-        type: "confirm",
-        name: "needsStorybook",
-        message: "Add Storybook for Visual Testing?",
-        default: true,
-        when: () => !!needsTesting,
-      },
-    ])
-  ).needsStorybook;
-
-  const needsE2eTesting = (
-    await inquirer.prompt<{ needsE2eTesting?: boolean }>([
-      {
-        type: "confirm",
-        name: "needsE2eTesting",
-        message: "Add Playwright for End-To-End Testing?",
-        default: true,
-        when: () => needsTesting,
-      },
-    ])
-  ).needsE2eTesting;
-
-  const needsEslint = (
-    await inquirer.prompt<{ needsEslint?: boolean }>([
-      {
-        type: "confirm",
-        name: "needsEslint",
-        message: "Add ESLint for Code Linting?",
-        default: true,
-        when: () => needsTesting,
-      },
-    ])
-  ).needsEslint;
-
-  const needsPrettier = (
-    await inquirer.prompt<{ needsPrettier?: boolean }>([
-      {
-        type: "confirm",
-        name: "needsPrettier",
-        message: "Add Prettier for Code Formatting?",
-        default: true,
-        when: () => needsTesting,
-      },
-    ])
-  ).needsPrettier;
-
-  const templateName = jsLibrary;
-  const TEMP_DIR = path.resolve(__dirname, "temp");
-
-  let packageObjs: Record<string, unknown> = {};
-
+const TEMP_DIR = path.resolve(__dirname, "temp");
+const EXCLUDE = [
+  "node_modules",
+  "vite.config.ts",
+  "package.json",
+  "tsconfig.json",
+  "README.md",
+];
+
+let packageObjs: Record<string, unknown> = {};
```

Comment:
To reduce function arguments, I've decided to turn `packageObjs` into a global variable.